### ISSUE TITLE
Fenced Code Blocks / CommonMark

### ISSUE THREAD

**Issue by huettenhain** [miyuchina/mistletoe#15]
Given the following markdown document:

    This
    ```
    code
    block
    ```
    does not convert properly.

Then the output of `md2jira` (for example) will be the following:

    This
    {{`}}
    code
    block
    {{`}}
    does not convert properly.

I am well-aware that triple backtick code blocks are not the original Markdown standard, but they have made it into [common mark](http://spec.commonmark.org/0.28/#fenced-code-blocks) and I consider them very useful. 

For me personally, this might be the only feature from common mark that I am missing, but would you be generally interested in implementing all of common mark or accept pull requests to that end?

**Comment by miyuchina** [comment]
Ah, it's a bug in `Paragraph.read` method actually. I'll fix it hopefully today.

mistletoe does support code fences. If you add a newline after "this":

    This

    ```
    code
    block
    ```
    does not convert properly.

This will convert to:

```html
<p>This
</p>
<pre>
<code>
code
block
</code>
</pre>
<p>does not convert properly
</p>
```

The missing newline is tripping things up.

CommonMark is a thorny issue that has been on my mind for a while. I'll open another issue to make my thoughts clear, but to answer your question for now, yes, I do plan to at least work towards a CommonMark-compliant implementation.

**Comment by miyuchina** [comment]
Should be fixed in c51b564. Let me know if there's any other problem! Closing for now.
